services/blockchain-indexer/config.js

```diff
@@ -213,4 +213,8 @@ config.estimateFees = {
 	bufferBytesLength: Number(process.env.ESTIMATES_BUFFER_BYTES_LENGTH) || 0,
 };
 
+config.invokeAllowedMethods = process.env.INVOKE_ALLOWED_METHODS
+	? String(process.env.INVOKE_ALLOWED_METHODS).split(',')
+	: [];
```

Comment:
If we are not defining defaults here, can we define endpoints required in .env and ecosystem file?

Reply:
As suggested [earlier](https://github.com/LiskHQ/lisk-service/pull/1972#discussion_r1415887303), please set the defaults everywhere (config files, .env etc..), as applicable.

Reply:
@vardan10, for this one setting to nothing will be the default. If you are referring to the methods used by fronend apps, maybe it would better to not mention those here and keep it generic as service can be used by other frontend clients.

Reply:
@sameersubudhi we had faced a bug after applying that comment and changed to current logic. I had already added `INVOKE_ALLOWED_METHODS=''` as commented value to `example.env`, `ecosystem.config.js` and `ecosystem.jenkins.config.js`. Please let me know if this needs to be added somewhere else as well.

Reply:
Added default values 👍